src/main/java/net/mcreator/element/types/interfaces/IColors.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * MCreator (https://mcreator.net/)
+ * Copyright (C) 2012-2020, Pylo
+ * Copyright (C) 2020-2023, Pylo, opensource contributors
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package net.mcreator.element.types.interfaces;
+
+import net.mcreator.ui.init.L10N;
+
+import java.util.List;
+
+@SuppressWarnings("unused") public interface IColors {
+	default List<ColorEntry> colorEntries(String description, String key) {
+		return List.of(new ColorEntry("blue", description, key), new ColorEntry("white", description, key),
```

Comment:
I have a simple question for you.
Why did you use a Preference entry class for your feature? I'm honestly very curious to know 😅